src/typescript/frontend/src/components/pages/launch-emojicoin/EmojiRain.tsx

```diff
@@ -0,0 +1,100 @@
+import { type symbolBytesToEmojis } from "@sdk/emoji_data";
+import { type AnimationPlaybackControls, useAnimate } from "framer-motion";
+import { useEffect, useMemo, useState } from "react";
+import { useWindowSize } from "react-use";
+
+const DROP_FALL_TIME = 7.5;
+const DROP_INTERVAL = 1;
+
+export function EmojiRainDrop({
+  name,
+  onClick,
+  index,
+  emojis,
+}: {
+  name: ReturnType<typeof symbolBytesToEmojis>;
+  onClick?: (data: ReturnType<typeof symbolBytesToEmojis>) => void;
+  index: number;
+  emojis: number;
+}) {
+  const { height, width } = useWindowSize();
+
+  // Position all emojis above the visible space.
+  const initialX = -200;
+
+  // Calculate a random position on the Y axis.
+  const y = useMemo(() => Math.random() * (width - 20) + 10, [width]);
```

Comment:
I think you have the variable names backwards? this is technically the initial `Y` position.

Also I think you should do this entirely with percentages and remove the window size hook- you can use `${percentage}vw` and `${percentage}vh` for the style